### PR TITLE
feat: violating gitlab repo by default

### PR DESCRIPTION
**Pull request by Benjie-legit**
#### What's being changed?

violating gitlab repo by default
#### Is this PR related to an existing issue?

https://github.com/Legit-Labs/legitify/issues/84
#### Check off the following:

- [X] This PR follows the CONTRIBUTION.md guidelines
- [X] I have self-reviewed my changes before submitting the PR

### CODE REVIEW

File: policies/gitlab/repository.rego [hunk 1]
```diff
@@ -54,11 +47,11 @@ project_has_too_many_admins {
...
 #   severity: LOW
 #   threat: Forked repositories may leak important code assets or sensitive secrets embedded in the code to anyone outside your organization, as the code becomes publicy-accessible
-default forking_allowed_for_repository = false
+default forking_allowed_for_repository = true
 
-forking_allowed_for_repository {
-	input.public == false
-	input.forking_access_level == "enabled"
+forking_allowed_for_repository = false {
+	input.public
```
Comment (noamd-legit) [inline policies/gitlab/repository.rego:53]: this policy applies to private repositories

File: policies/gitlab/repository.rego [hunk 2]
```diff
@@ -86,11 +79,8 @@ missing_default_branch_protection {
 #   threat: Rewriting project history can make it difficult to trace back when bugs or security issues were introduced, making them more difficult to remediate.
 default missing_default_branch_protection_force_push = true
 
-missing_default_branch_protection_force_push {
-	missing_default_branch_protection
-}
-
 missing_default_branch_protection_force_push = false {
+	not missing_default_branch_protection
```
Comment (noamd-legit) [inline policies/gitlab/repository.rego:83]: are you sure it's needed?

File: policies/gitlab/repository.rego [hunk 3]
```diff
@@ -105,15 +95,12 @@ missing_default_branch_protection_force_push = false {
...
-default repository_require_code_owner_reviews_policy = false
+default repository_require_code_owner_reviews_policy = true
 
-repository_require_code_owner_reviews_policy {
-	missing_default_branch_protection
-}
-
-repository_require_code_owner_reviews_policy {
+repository_require_code_owner_reviews_policy = false {
+	not missing_default_branch_protection
```
Comment (noamd-legit) [inline policies/gitlab/repository.rego:101]: needed?

File: policies/gitlab/repository.rego [hunk 4]
```diff
@@ -170,15 +157,13 @@ no_conversation_resolution {
...
 #   severity: LOW
 #   threat: A commit containing malicious code may be crafted by a malicious actor that has acquired write access to the repository to initiate a supply chain attack. Commit signing provides another layer of defense that can prevent this type of compromise.
-default no_signed_commits = false
+default no_signed_commits = true
 
-no_signed_commits {
-	input.push_rules.reject_unsigned_commits == false
+no_signed_commits = false {
+	input.push_rules.reject_unsigned_commits
+	not is_null(input.push_rules)
```
Comment (noamd-legit) [inline policies/gitlab/repository.rego:164]: not needed

File: policies/gitlab/repository.rego (hunk 3 above)
Comment (Benjie-legit) [reply policies/gitlab/repository.rego]: fixed

File: policies/gitlab/repository.rego (hunk 2 above)
Comment (Benjie-legit) [reply policies/gitlab/repository.rego]: fixed

File: policies/gitlab/repository.rego (hunk 1 above)
Comment (Benjie-legit) [reply policies/gitlab/repository.rego]: fixed

**Review by noamd-legit — APPROVED** [review]

**Pull request merged**